sklearn/metrics/classification.py

```diff
@@ -1530,6 +1530,12 @@ class 2       1.00      0.67      0.80         3
     else:
         labels = np.asarray(labels)
 
+    # labelled micro average
+    label_micro_average = False
+    if y_type.startswith('multiclass') and \
```

Comment:
I think `== 'multiclass'` should be more correct... but you also need to check for binary.

Reply:
done